frontend/src/app/features/calendar/te-calendar/te-calendar.component.ts

```diff
@@ -202,63 +223,83 @@ export class TimeEntryCalendarComponent {
     private sanitizer:DomSanitizer,
     private configuration:ConfigurationService,
     private timezone:TimezoneService,
-    private timeEntryEdit:TimeEntryEditService,
-    private timeEntryCreate:TimeEntryCreateService,
     private schemaCache:SchemaCacheService,
     private colors:ColorsService,
     private browserDetector:BrowserDetector,
     private calendar:OpCalendarService,
     readonly weekdayService:WeekdayService,
     readonly dayService:DayResourceService,
-  ) {}
+    readonly turboRequests:TurboRequestsService,
+    readonly pathHelper:PathHelperService,
+  ) { }
+
+  ngAfterViewInit():void {
+    document.addEventListener('dialog:close', this.closeDialogHandler);
+  }
+
+  ngOnDestroy():void {
+    document.removeEventListener('dialog:close', this.closeDialogHandler);
+  }
 
-  async requireNonWorkingDays(start:Date|string, end:Date|string) {
-    this.nonWorkingDays = await firstValueFrom(this.dayService.requireNonWorkingYears$(start, end));
+  async requireNonWorkingDays(start:Date | string, end:Date | string) {
+    this.nonWorkingDays = await firstValueFrom(
+      this.dayService.requireNonWorkingYears$(start, end),
+    );
   }
 
   public calendarEventsFunction(
     fetchInfo:EventSourceFuncArg,
     successCallback:(events:EventInput[]) => void,
     failureCallback:(error:Error) => void,
-  ):void|PromiseLike<EventInput[]> {
+  ):void | PromiseLike<EventInput[]> {
     const start = moment(fetchInfo.startStr);
     const end = moment(fetchInfo.endStr);
     void this.fetchTimeEntries(start, end)
       .then(async (collection) => {
         this.entries.emit(collection);
 
-        successCallback(await this.buildEntries(collection.elements, fetchInfo));
+        successCallback(
+          await this.buildEntries(collection.elements, fetchInfo),
+        );
       })
       .catch(failureCallback);
   }
 
-  protected fetchTimeEntries(start:Moment, end:Moment):Promise<CollectionResource<TimeEntryResource>> {
-    if (!this.memoizedTimeEntries
+  protected fetchTimeEntries(
+    start:Moment,
+    end:Moment,
+  ):Promise<CollectionResource<TimeEntryResource>> {
+    if (
+      !this.memoizedTimeEntries
       || this.memoizedTimeEntries.start.valueOf() !== start.valueOf()
-      || this.memoizedTimeEntries.end.valueOf() !== end.valueOf()) {
+      || this.memoizedTimeEntries.end.valueOf() !== end.valueOf()
+    ) {
       const promise = firstValueFrom(
-        this
-          .apiV3Service
-          .time_entries
-          .list({ filters: this.dmFilters(start, end), pageSize: 500 }),
-      )
-        .then((collection) => {
-          this.memoizedCreateAllowed = !!collection.createTimeEntry;
-
-          return collection;
-        });
+        this.apiV3Service.time_entries.list({
```

Comment:
Same here, these blocks are all reformatted without change. That makes it harder to reason about the changes. Does your editor have a "reformat only changed lines" option?

Reply:
It's reformatting the entire file. But I am wondering why eslint is not complaining about this as being wrong after the autofix? It seems we have ambiguous settings here.